cmd/agent/flow.go

```diff
@@ -119,6 +129,7 @@ func runFlow() error {
 
 		r.HandleFunc("/-/reload", func(w http.ResponseWriter, _ *http.Request) {
 			err := reload()
+			ready = err == nil
```

Comment:
This would need a mutex to protect ready or to use an atomic bool. Can we just have it print "ready" for now instead of trying to slowly add in health checks?

Reply:
Will use atomic bool.